webui/ocw/lib/db.py

```diff
@@ -21,7 +21,7 @@ def update_or_create_instance(provider, instance_id, region, csp_info, active=Tr
         o.age = t_now - o.first_seen
         o.active = active
         o.csp_info = json.dumps(csp_info, ensure_ascii=False)
-        if o.state == StateChoice.UNK:
+        if o.state != StateChoice.DELETING:
```

Comment:
and if it's DELETED ? 
and if it is DELETING why we saving the state ?

Reply:
Add a  warning, if we change from DELETED to something else